Gaijinizer patch/Map122.txt

```diff
@@ -461,63 +481,63 @@ dog?#MTLed
 だけど、代わりに自由気ままに暴れて、喰ってやった。
  隙を見せた人間から、次々とね…
 > CONTEXT: Map122/events/5/pages/1/196/Dialogue/enemy14
-Well, I did cooperate with the "experiment".
-But instead, I went wild and ran wild.
-From people who showed a chance, one after
-another ...#MTLed
+I cooperated with their experiments, but in
+exchange they let me run rampant and eat whoever
+I wanted. Anyone who showed me the slightest bit
+of weakness was eaten, one by one...#RonIffy
 > END STRING
 
 > BEGIN STRING
 ガラス越しのあの人間どもの恐怖や嫌悪に満ちた表情…
 
 アンタにも見せてやりたかったねぇ…♡
 > CONTEXT: Map122/events/5/pages/1/201/Dialogue/enemy14
-Afraid and hateful expressions of those humans
-through glass...I wanted to show it to you too
-...#MTLed
+The looks of fear and disgust on those human's
+faces behind the glass...
+
+I wish you could have seen it...♡#Ron
 > END STRING
 
 > BEGIN STRING
 ……………ああ、だけど……
 
 一人、変な奴がいたっけか………
 > CONTEXT: Map122/events/5/pages/1/207/Dialogue/enemy14
-...... Oh, but ...... One person, strange guy or
-not .........#MTLed
+Oh, but there was this one weirdo there...#Ron
 > END STRING
 
 > BEGIN STRING
 変な奴…？
 > CONTEXT: Map122/events/5/pages/1/211/Dialogue/kazuya
-Funny guy ...?#MTLed
+Weirdo...?#Ron
 > END STRING
 
 > BEGIN STRING
 組織のエライやつみたいな、茶髪の女だったんだけど
 
  そいつだけは、私に拍手を送って、笑顔で言ったよ。
 > CONTEXT: Map122/events/5/pages/1/216/Dialogue/enemy14
-It was a woman with a brown hair, like an
-organized elie. The only thing that applauded me
-and said with a smile.#MTLed
+She was this brown-haired woman, looked like she
+was one of the higher-ups. She'd clap and smile
+while talking to me.#Ron
 > END STRING
```

Comment:
This is a complete localization, but I'd prefer "reddish-haired" over "brown-haired" just so that it's a bit more obvious for the western audience.